charters/UNITY/charter.md

```diff
@@ -0,0 +1,54 @@
+# Unity Benchmarking Service
+
+## Description
+
+The Unity benchmarking service automates the process of running and benchmarking pipelines.
+
+## Objectives
+
+The Unity benchmarking service aims to automate benchmarking for HCA pipelines, enforce standards for consistent reporting for historic comparison and enable the public to participate in suggesting improvements to pipelines by empowering them to implement improvements and test results using the same infrastructure as the HCA data processing pipelines team. The Unity benchmarking service will be used by those making pipelines for the HCA data processing service and will be public, so anyone can benchmark their implemented ideas against HCA pipelines.
+
+## In-scope
+
+* Benchmarking for HCA pipelines will be automated and included in Unity
+* Benchmarking data will be available through Unity but not controllable by the users
+* Benchmarking data can change without notice but will be consistent in format and schema with previous data or notice will be given to users
+* Leaderboards showing how pipelines differ will be created as different pipelines for the same assay are collected
+* Documentation on how to use Unity and the assumptions pipelines must meet to be ran using Unity
+* Documentation on benchmarking process and outputs
+
+### Scientific "guardrails"
+
+* Benchmarking pipelines are reviewed by the HCA project and treated as data processing pipelines (engineering updates do not require HCA oversight; scientific updates do).
+* Metrics and how metrics are used in the leaderboards are reviewed by the HCA project.
+
+## Out-of-scope
+
+* Educational training or user assistance on how to create pipelines
+* Recommendations on how to improve pipelines based on benchmarking
+* Benchmarking community pipelines are not required to be included in the Unity benchmarking service (but are allowed)
```

Comment:
_Benchmarking ... **is** not required ... but **is** allowed ..._
Can this be clarified? This team will not benchmark community pipelines, but the DCP community may?

Reply:
We are not obligated to set up benchmarking for every assay everyone wants to submit to benchmark against. It would be too much work. If we can leverage what we have for standard pipelines then community pipelines can just benchmark using what is in the system. But for a new assay, if the pipeline is in our roadmap then we are more likely to set up the assay in the system for community pipelines. (We have to find reference data to run, potentially define outputs, potentially initiate benchmarking pipelines. Depending on how proactive the community members are, this may be a significant task). I will try to clarify.